### PR TITLE
Release v1.12.1

### PR DESCRIPTION
**Pull request by priyawadhwa**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
 
+## Version 1.12.1 - 2020-07-17
+
+Features:
+* Add support for Calico CNI (--cni=calico) [#8571](https://github.com/kubernetes/minikube/pull/8571)
+
+Bug Fixes:
+* Fix bugs which prevented upgrades from v1.0+ to v1.12 [#8741](https://github.com/kubernetes/minikube/pull/8741)
+* Add KicBaseImage to existing config if missing (fixes v1.9.x upgrade) [#8738](https://github.com/kubernetes/minikube/pull/8738)
+* add required quote to alias kubectl [#8747](https://github.com/kubernetes/minikube/pull/8747)
```
Comment (tstromberg) [inline CHANGELOG.md:11]: This was a docs only change

Comment (priyawadhwa) [reply CHANGELOG.md]: done

Comment (priyawadhwa) [reply CHANGELOG.md]: thanks!

**Pull request merged**